packages/database/src/drizzle/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from "./auth.js";
+export * from "./auth";
```

Comment:
![image](https://github.com/ari-becker/drizzle-1558-repro/assets/1388505/103b44c2-c0d3-47e6-b03d-29d49885febb)

@Angelelz the extension is necessary when using `node16`

Reply:
(if you changed the `moduleResolution` somewhere, it's not in the PR...?)

Reply:
Forgot to push that one. It's updated